README.md

```diff
@@ -1,5 +1,10 @@
-# clarity
+# Clarity
 
 [![Build Status](https://travis-ci.org/althea-mesh/clarity.svg?branch=master)](https://travis-ci.org/althea-mesh/clarity)
+[![Latest Version](https://img.shields.io/crates/v/toml.svg)](https://crates.io/crates/clarity)
```

Comment:
Shouldn't it link to `https://img.shields.io/crates/v/clarity.svg` instead of linking to toml crate?

Reply:
Yes, I stole those from Alex Crichton's `toml` package